SystemTest/main.go

```
@@ -1,50 +1,214 @@
+	err = json.Unmarshal(all, &result)
+	if err != nil {
+		return nil, err
 	}
-	fmt.Printf("status code was [%v]\n", get.StatusCode)
 	fmt.Printf("response is [%v]\n", string(all))
+	return &result, nil
+}
+
+func (s *ServiceClient) Login() (string, error) {
+	///product/items
+	url := createUrl(fmt.Sprintf("usr/login"))
```

golangci-lint (1.62.2)

S1039: unnecessary use of fmt.Sprintf

(gosimple)